modules/aws/kube-etcd/variables.tf

```diff
@@ -95,9 +95,32 @@ EOF
 }
 
 variable "instance_config" {
-  description = "(Optional) Desired etcd nodes configuration."
-  type        = map(string)
-  default     = {}
+  description = "Desired etcd nodes configuration."
+  type        = object({
+    count              = number
+    image_id           = string
+    ec2_type           = string
+    root_volume_size   = number
+    data_volume_size   = number
+    data_device_name   = string
+    data_device_rename = string
+    data_path          = string
+  })
+}
+
+locals {
+  instance_config = merge({
+    count              = "1"
+    ec2_type           = "t3.medium"
+    root_volume_size   = "40"
+    data_volume_size   = "100"
+    data_device_name   = "/dev/sdf"
+    data_device_rename = "/dev/nvme1n1"
+    data_path          = "/etcd/data"
+
+    // CoreOS Container Linux stable 2512.3.0
+    image_id = "ami-0c45c2b94700c3e25"
+  }, var.instance_config)
```

Comment:
These lines seem not use on this module, we can remove.

Reply:
thank for the reviewing, I have removed the unused local variable you mentioned above